src/dags/deelmobiliteit.py

```diff
@@ -0,0 +1,268 @@
+
+
+def quote(instr: str) -> str:
+    """needed to put quotes on elements in geotypes for SQL_CHECK_GEO"""
+    return f"'{instr}'"
+
+
+with DAG(
+    dag_id,
+    description=description,
+    default_args=default_args,
+    user_defined_filters=dict(quote=quote),
+) as dag:
+
+    # 1. Post message on slack
+    slack_at_start = MessageOperator(
+        task_id="slack_at_start",
+        http_conn_id="slack",
+        webhook_token=slack_webhook_token,
+        message=f"Starting {dag_id} ({DATAPUNT_ENVIRONMENT})",
+        username="admin",
+    )
+
+    # 2. Load scooter data into DB
+    import_scooter_data = PythonOperator(
+        task_id="import_scooter_data",
+        python_callable=import_scooter_data,
+        op_kwargs=dict(
+            table_name=f"{dag_id}_scooters_new",
+            fleyx_api_endpoint=f"{fleyx_base_url}{endpoint_scooters['fleyx']}",
+            fleyx_api_header={
+                "content-type": "application/json",
+                "x-api-key": fleyx_api_key,
+            },
+            ridecheck_token_endpoint=f"{ridecheck_token_url}",
+            ridecheck_token_payload={
+                "grant_type": "client_credentials",
+                "client_id": ridecheck_token_client_id,
+                "client_secret": ridecheck_token_client_secret,
+                "scope": f"{ridecheck_base_url}/scooter.read",
+            },
+            ridecheck_token_header={"content-type": "application/x-www-form-urlencoded"},
+            ridecheck_data_endpoint=f"{ridecheck_base_url}{endpoint_scooters['ridecheck']}",
+            ridecheck_data_header={
+                "content-type": "application/json",
+            },
+        ),
+    )
+
+    # 3. Load auto data into DB
+    import_auto_data = PythonOperator(
+        task_id="import_auto_data",
+        python_callable=import_auto_data,
+        op_kwargs=dict(
+            table_name=f"{dag_id}_autos_new",
+            mywheels_api_endpoint=f"{mywheels_base_url}{endpoint_autos['mywheels']}",
+            mywheels_api_header={
+                "content-type": "application/json",
+                "X-Simple-Auth-App-Id": mywheels_api_key,
+            },
+            mywheels_api_payload={
+                "jsonrpc": "2.0",
+                "id": 1,
+                "method": "search.map",
+                "params": {
+                    "locationPoint": {
+                        "latitudeMin": 50,
+                        "latitudeMax": 54,
+                        "longitudeMin": 3,
+                        "longitudeMax": 8,
+                    },
+                    "timeFrame": {"startDate": None, "endDate": None},
+                },
+            },
+        ),
+    )
+
+    # 4. Dummy operator acts as an Interface between parallel tasks
+    # to another parallel tasks (i.e. lists or tuples) with different number
+    # of lanes (without this intermediar, Airflow will give an error)
+    Interface = DummyOperator(task_id="interface")
```

Comment:
Ik zie nu in Airflow 2 talloze warnings over de aanpak met de Interface, geeft veel ruis in de logging. Vlgs. mij bouwen we de DAG op een goede manier op, maar wellicht zien we toch iets over het hoofd. Of, het is weer een Airflow bugjes.

Reply:
Ok ik zal eens kijken, zie lokaal zelf een errors